### DISCUSSION TITLE
can we replace the engine of nebula

### DISCUSSION THREAD

**Discussion by Gibson1314** [vesoft-inc/nebula#4039] [Q&A]
nebula 的引擎是 RocksDB。
fork 自 RocksDB，在 rockside 增加了 SidePlugin 插件体系的 ToplingDB 可否替代 RocksDB？

**Comment by SuperYoko** [comment]
只要接口一致，应该是可以的，但nebula目前是静态编译的，所以至少要从编译层面来搞替代。
